### PR TITLE
Image from asset manager

### PR DESCRIPTION
**Pull request by ssatish**
Image field on campaign should be editable. We are using asset manager to store the images uploaded. And the asset url must be captured in the content store. This PR is to change govuk-content-schemas to reflect this change.

### CODE REVIEW

File: formats/financial_releases_campaign/publisher/details.json [hunk 1]
```diff
@@ -25,5 +25,28 @@
     "footer_legal_disclaimer": {
       "$ref" : "#/definitions/multiple_content_types"
     }
+  },
+  "definitions": {
+    "asset_link": {
```
Comment (boffbowsh) [inline formats/financial_releases_campaign/publisher/details.json]: These should go in the base `definitions.json` file

File: jmeter.log [hunk 2]
```diff
@@ -0,0 +1,36 @@
+2015/12/23 14:20:35 INFO  - jmeter.util.JMeterUtils: Setting Locale to en_US
```
Comment (boffbowsh) [inline jmeter.log]: You've left a log file in here

File: dist/formats/financial_releases_campaign/publisher_v2/schema.json [hunk 3]
```diff
@@ -258,6 +258,27 @@
...
+        "url",
+        "content_type",
+        "alt_text"
+      ],
+      "properties": {
+        "url": {
+          "type": "string",
+          "format": "uri"
+        },
+        "content_type": {
```
Comment (boffbowsh) [inline dist/formats/financial_releases_campaign/publisher_v2/schema.json]: Why is the content type needed?

File: formats/financial_releases_campaign/publisher/details.json (hunk 1 above)
Comment (ssatish) [inline formats/financial_releases_campaign/publisher/details.json]: @boffbowsh What is the file path for this base definitions.json file ?

File: dist/formats/financial_releases_campaign/publisher_v2/schema.json (hunk 3 above)
Comment (ssatish) [inline dist/formats/financial_releases_campaign/publisher_v2/schema.json]: @boffbowsh This format will have a pdf document uploaded to it too. Very soon. Does that mean we will need to have content type then ?  
If not, I can remove it. I am not really using the content type in the app.

Comment (ssatish) [inline dist/formats/financial_releases_campaign/publisher_v2/schema.json]: I have removed content type

File: formats/financial_releases_campaign/publisher/details.json (hunk 1 above)
Comment (boffbowsh) [inline formats/financial_releases_campaign/publisher/details.json]: https://github.com/alphagov/govuk-content-schemas/blob/master/formats/definitions.json

**Comment by boffbowsh** [comment]
This needs a rebase

**Comment by ssatish** [comment]
@boffbowsh Rebased.

**Comment by boffbowsh** [comment]
You still need to address my comment from previously. Can you also please add some more context to this? Either to the PR or the commit messages?

**Comment by ssatish** [comment]
@boffbowsh I don't really understand your comment regarding asset_link to go into definitions file.

**Comment by boffbowsh** [comment]
You've created a new definition in that file. You'll see that the `image` type is defined in the base `definitions.json` file, which is also where the `asset_link` type should be defined so that other formats can share it.

**Comment by boffbowsh** [comment]
Based on the PR description, it seems like this will only store images, is this correct? If so, what was wrong with the original `image` type?

**Comment by ssatish** [comment]
Maybe I don't need this PR. Because the whole upload functionality does seem to work without this change merged.

**Comment by ssatish** [comment]
I will close the PR.

**Pull request closed without merging**